.appveyor.yml

```diff
@@ -59,28 +66,38 @@ platform: x86
 
 # scripts to run before build
 before_build:
-  - set PATH=%QT%;%COMPILER%;%PATH%
+  - ps: $env:Path = "$env:QT;$env:COMPILER;$env:Path"
+  - sh: |
+      brew install qt jack p7zip
+      export PATH="/usr/local/opt/qt/bin:$PATH"
+      export JACKLIB="/usr/local/opt/jack/lib"
+      export JACKINC="/usr/local/opt/jack/include"
   - cd BambooTracker
 
 # to run your custom scripts instead of automatic MSBuild
 build_script:
-  - qmake BambooTracker.pro CONFIG+=release CONFIG-=debug
-  - mingw32-make
-  - md bin
-  - copy release\BambooTracker.exe bin
-  - cd bin
-  - windeployqt BambooTracker.exe
-  - ren translations lang
-  - cd ..
-  - copy .qm\*.qm bin\lang
+  - ps: |
+      qmake.exe BambooTracker.pro CONFIG+=debug CONFIG+=console CONFIG+=nostrip
+      & $env:MAKE -j2
+      # Fixes AppVeyor build erroring out...?
+      echo "=== BUILDING DONE ==="
```

Comment:
I don't know *why*, but this `echo` here keeps the build from getting stuck without log output after `mingw32-make` and miraculously erroring out. [(Example here.)](https://ci.appveyor.com/project/OPNA2608/bambootracker/builds/32060738/job/97bgde1lwc07axqe) Maybe AppVeyor thinks it got stuck, or PowerShell does something weird, I don't know.